content/tokio/tutorial/spawning.md

```diff
@@ -16,12 +16,12 @@ Then create a new, empty `src/main.rs` and continue.
 
 # Accepting sockets
 
-The first thing our Redis server needs to do is accept inbound TCP sockets. This
-is done with [`tokio::net::TcpListener`][tcpl].
+The first thing our Redis server needs to do is to accept inbound TCP sockets.
+This is done with [`tokio::net::TcpListener`][tcpl].
 
 [[info]]
 | Many of Tokio's types are named the same as their synchronous equivalent in
-| the Rust standard library. When it makes sense, Tokio exposes the same APIs
+| the Rust standard library. When feasible, Tokio exposes the same APIs
 | as `std` but using `async fn`.
```

Comment:
I'm not convinced about this change.

Reply:
What about: When reasonable?

Reply:
I prefer the original phrasing, to be honest.